package.json

```
@@ -5,6 +5,7 @@
   "description": "CodeQL action",
   "scripts": {
     "build": "tsc",
+    "build-cli": "webpack --mode production",
     "test": "ava src/** --serial --verbose",
     "lint": "tslint -p . -c tslint.json 'src/**/*.ts'",
     "removeNPMAbsolutePaths": "removeNPMAbsolutePaths . --force"
```

Suggestion: add a `postinstall` step that builds the CLI automatically after a successful `npm install`. This will make it easier for new users to acquire the CLI by reducing the number of installation steps.

E.g.

```suggestion
  "scripts": {
    "build": "tsc",
    "build-cli": "webpack --mode production",
    "test": "ava src/** --serial --verbose",
    "lint": "tslint -p . -c tslint.json 'src/**/*.ts'",
    "removeNPMAbsolutePaths": "removeNPMAbsolutePaths . --force",
    "postinstall": "npm run-script build-cli"
```